migration/main.py

```diff
@@ -13,16 +14,43 @@
 from manager import AccountManagerFactory, CourseManager
 from utils import convert_csv_to_int_list, find_entity_by_id, time_execution
 
+summaryLogBuffer = StringIO()
+summaryLogHandler = logging.StreamHandler(summaryLogBuffer)
+summaryLogHandler.setLevel(logging.WARNING)
+
+logging.basicConfig(
+    level=logging.INFO,
```

Comment:
Might be worthwhile to make this log level configurable

Reply:
Looks like it might be configured already down below just might want to check if it still works to change the levels

Reply:
That level setting there was just to set the initial level of logging before the log level parameter is read.  It seems OK this way.